fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WebApplicationExceptionMapper.java

```diff
@@ -39,7 +39,7 @@
 
     @Override
     public Response toResponse(final WebApplicationException e) {
-        LOGGER.debug(
+        LOGGER.error(
```

Comment:
Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`